kogitoq/gadaq/src/main/java/life/genny/gadaq/live/data/InternalConsumer.java

```diff
@@ -127,11 +131,54 @@ public void getEvent(String event) {
 
 		log.info("Received Event : " + SecurityUtils.obfuscate(event));
 
+		// Quick search
+		if(filter.isQuickSearchDropdown(msg)) {
```

Comment:
This should go in the `Events.java` Class

Reply:
dropkick causes issues